modules/1plusXRtdProvider.js

```diff
@@ -216,14 +204,6 @@ export const updateBidderConfig = (bidder, ortb2Updates, bidderConfigs) => {
   return bidderConfigCopy;
 };
 
-const setAppnexusAudiences = (audiences) => {
```

Comment:
removing this means that appnexus won't get audience data; I'm not sure that's the intent. Airgrid is replacing this with another kind of appnexus logic that's slightly more general, but they are not removing it entirely.